.github/workflows/cmake.yml

```diff
@@ -348,15 +348,45 @@ jobs:
             packages:  ninja-build llvm-11-tools
             gcov-exec: llvm-cov-11 gcov
 
-          - name: Windows MSVC Win32
+          - name: Windows MSVS 2022 v143 Win32
             os: windows-latest
             compiler: cl
-            cmake-args: -A Win32
+            cmake-args: -G "Visual Studio 17 2022" -A Win32 -T v143
 
-          - name: Windows MSVC Win64
+          - name: Windows MSVS 2022 v143 Win64
```

Comment:
Should use `MSVC` as it is more accurate... ´VS` means Visual Studio and it is frontend for multiple programming languages. VC can mean either C, C++ or C#, but these three languages are handled by same set of compiler binaries, commonly known as Common Language (CL) compiler.

Reply:
Renamed.